Error while checking build/O2/fullCI for e4d550ca6d50df135304de0c3bf87ead417dbb12 at 2023-04-01 09:17:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/11083-slc8_x86-64/0/PWGCF/JCorran/TableProducer/JCatalyst.cxx:30:10: fatal error: DetectorsVertexing/DCAFitterN.h: No such file or directory
ninja: build stopped: subcommand failed.


## sw/BUILD/o2checkcode-latest/log
--
++ echo '========== List of errors found =========='
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
++ GRERR=1
++ [[ 1 == 0 ]]
++ mkdir -p /sw/INSTALLROOT/783f9ca8f2b4d8f502abb7904a29724575161963/slc8_x86-64/o2checkcode/1.0-local7/etc/modulefiles
++ cat
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11083/e4d550ca6d50df135304de0c3bf87ead417dbb12/build_O2_fullCI/pretty.html).

--- a/Detectors/EMCAL/reconstruction/include/EMCALReconstruction/RecoContainer.h
+++ b/Detectors/EMCAL/reconstruction/include/EMCALReconstruction/RecoContainer.h
@@ -254,7 +254,7 @@ class RecoContainerReader
   EventContainer& nextEvent();
 
   /// \brief Check whehter there are more events in the container
-  /// \return True if the event is not the last event, false otherwise
+  /// \return True if the event is not the last event, false otherwise.
   bool hasNext() const { return mCurrentEvent < mOrderedInteractions.size(); }
 
   /// \brief Get the number of events in the container

